packages/selection/backend/sirius-components-collaborative-selection/src/test/java/org/eclipse/sirius/components/collaborative/selection/SelectionEventProcessorTests.java

```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.collaborative.selection;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.function.Predicate;
+
+import org.eclipse.sirius.components.collaborative.api.ChangeDescription;
+import org.eclipse.sirius.components.collaborative.api.ChangeKind;
+import org.eclipse.sirius.components.collaborative.api.IRepresentationRefreshPolicyRegistry;
+import org.eclipse.sirius.components.collaborative.representations.SubscriptionManager;
+import org.eclipse.sirius.components.collaborative.selection.dto.SelectionEventInput;
+import org.eclipse.sirius.components.collaborative.selection.dto.SelectionRefreshedEventPayload;
+import org.eclipse.sirius.components.core.api.IEditingContext;
+import org.eclipse.sirius.components.core.api.IInput;
+import org.eclipse.sirius.components.core.api.IObjectService;
+import org.eclipse.sirius.components.core.api.IPayload;
+import org.eclipse.sirius.components.selection.description.SelectionDescription;
+
+import reactor.test.StepVerifier;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Unit tests of the SelectionEventProcessor.
+ *
+ * @author mcharfadi
+ */
+public class SelectionEventProcessorTests {
```

Comment:
Don't add more unit tests, contribute to the integration tests if needed, there's already a test suite for that

Reply:
Done